apps/web/lib/zod/schemas/import-csv.ts

```diff
@@ -1,11 +1,12 @@
 import { z } from "zod";
+import { parseDateSchema } from "./utils";
 
 export const linkMappingSchema = z.object({
   domain: z.string(),
   url: z.string(),
   key: z.string(),
-  createdAt: z.string().optional(),
-  tags: z.string().optional(),
-  title: z.string().optional(),
-  description: z.string().optional(),
+  createdAt: parseDateSchema.nullish(),
```

Comment:
I think this should still just be a string - it's the mapping to a CSV column header, not a data value

Reply:
yup